core/new-gui/src/app/dashboard/user/type/dashboard-project.interface.ts

```diff
@@ -5,4 +5,5 @@ export interface DashboardProject {
   ownerID: number;
   creationTime: number;
   color: string | null;
+  writeAccess: boolean;
```

Comment:
we have `entry.workflow.accessLevel` for workflows, can we align the same for all resources?

Reply:
Changed.